docs/devel/releasing.md

```diff
@@ -216,6 +217,15 @@ Finally, from a clone of upstream/master, *make sure* you still have
 `RELEASE_VERSION` set correctly, and run `./build/mark-stable-release.sh
 ${RELEASE_VERSION}`.
 
+### Updating the master branch
+
+If you are cutting a new release series, please also update the master branch:
+change the `latestReleaseBranch` in `cmd/mungedocs/mungedocs.go` to the new
+release branch (`release-X.Y`), run `hack/update-generated-docs.sh`. This will
+let the unversioned warning in docs point to the latest release series. Please
+send the changes as a PR titled "Update the latestReleaseBranch to release-X.Y
+in the munger".
+
```

Comment:
@ihmccreery I addressed most of your comments, but didn't specify "whom to assign it to", because I think the change is fairly straightforward. PTAL. Thanks!